inflation/dataset/reader.py

```diff
@@ -185,7 +187,7 @@ def read(self, filename, fields=None) -> InflationDataset:
                         date,
                     )
                     records.append(record)
-        print(total_pages, item_pages)
+        # print(total_pages, item_pages)
```

Comment:
It is still useful to see total. Could you remove other comments but could you log.info the result at the end?

Reply:
I added it.